docs/design/design-reviews/trade-studies/template.md

```diff
@@ -66,6 +68,18 @@ If applicable, describe the boundaries from which we have to design the solution
 - Backward Compatibility. For example, public interfaces consumed by client or third party apps cannot introduce breaking changes.
 - Integrations or dependencies with other systems. For example, push notifications to client apps must be done via existing websockets channel.
 
+#### Accessibility
```

Comment:
This is a fantastically written section, thank you so much for taking the time to write all this out, the content is top notch. To me, I think that something as important as Accessibility should go before the optional sections.

That said, I also think that this section should match the template in style a little more. Namely, the other sections give suggestions as lists. Breaking down the paragraph here would match the rest of the content. What do you think?

Reply:
Yeah let me give this change a shot and see what you think